RELEASE_NOTES.md

```diff
@@ -1,111 +1,159 @@
 # Release Notes
 
-## 2.1.1 "Millenium Falcon" - TBD
+## 2.1.1 "Millenium Falcon" - Mar 27, 2020
 
-* This release introduces support for Python 3 (Python 3.5 and higher). It means the agent now
-  supports Python 2.6, 2.7 and >= 3.5.
+This release includes significant changes to the Scalyr Agent code base.  The release highlights
+include:
 
-  On package installation, the installer will try to find a suitable Python version which will
-  be used for running the agent. It defaults to Python 2 (``/usr/bin/env python2``), but if a
-  suitable Python 2 version is not found, it will try to use Python 3 if available (
-  ``/usr/bin/env python3``).
+* Support for Python 3 (Python 3.5 and higher) in addition to existing support for Python 2.6
+  and 2.7.  See below for instructions on how to control which version of Python is used by
+  the Scalyr Agent.
+* No backward incompatiable changes have been made with the exception of the default K8s
```

Comment:
This one sentence seems a bit hard to read but I can't think of a way to make it better

Reply:
Thanks for the callout.  I did rework the sentence.. I think it's easier to read now.